solana-programs/anchor/audius-data/programs/audius-data/src/lib.rs

```diff
@@ -582,6 +655,13 @@ pub struct UserAuthorityDelegate {
     pub user_storage_account: Pubkey,
 }
 
+/// Authority delegation status account
+#[account]
+pub struct AuthorityDelegationStatus {
+    // Revoke status of an authority's delegation
```

Comment:
kinda, i think it should include status because it's a comment for a bool not a function so it shouldn't be an action.
i'll change it to 
```
// Revoke status for an authority's delegation eligibility
```

Reply:
ahh gotcha! sg